dna/dnabackup.json

```diff
@@ -0,0 +1,186 @@
+  "generator": "hc-scaffold:0.0.2+afbc120",
+  "Version": 1,
+  "UUID": "2ef9fa48-2a33-424f-a4bf-b0591b2a71d8",
+  "Name": "holodex",
+
+  "Properties": {
+    "description": "A pluggable app for other holochain based apps for enabling indexing",
+    "language": "en"
+  },
+
+  "PropertiesSchemaFile": "properties_schema.json",
+  "DHTConfig": {
+    "HashType": "sha2-256"
+  },
+  "BasedOn": {
+      "H": null
+  },
+  "Zomes": [
+  {
+      "Name": "anchor",
+      "Description": "Anchor zome",
+      "RibosomeType": "js",
+      "CodeFile": "anchor.js",
+      "Entries": [
+          {
+              "Name": "anchor",
+              "DataFormat": "json",
+              "Sharing": "public"
+          },
+          {
+              "Name": "anchor_links",
+              "DataFormat": "links",
+              "Sharing": "public"
+          }
+      ],
+      "Functions": [
+          {
+              "Name": "anchor_type_create",
+              "CallingType": "string",
+              "Exposure":"public"
+          },
+          {
+            "Name": "addAnchor",
+            "CallingType": "string",
+            "Exposure":"public"
+          },
+          {
+            "Name": "anchor_create",
+            "CallingType": "json",
+            "Exposure":"public"
+          },
+          {
+            "Name": "anchor_update",
+            "CallingType": "json",
+            "Exposure":"public"
+          },
+          {
+            "Name": "anchor_type_list",
+            "CallingType": "string",
+            "Exposure":"public"
+          }
+
+
+        ]
+    },
+    {
+
+      "Name": "ConfigureIndexNodes",
+      "Description": "Providing index nodes configurations",
+      "NucleusType": "js",
+      "CodeFile": "configureindexnodes.js",
+      "Entries": [
+        {
+          "Name": "NumberOfNodes",
+          "Required": true,
+          "DataFormat": "string",
+          "Sharing": "public",
+          "_": "cru"
+        }
+      ],
+      "Functions": [
+        {
+          "Name": "NumberOfNodesCreate",
+          "CallingType": "string",
+          "Exposure": "public",
+          "_": "c:NumberOfNodes"
+        },
+        {
+          "Name": "NumberOfNodesRead",
+          "CallingType": "string",
+          "Exposure": "public",
+          "_": "r:NumberOfNodes"
+        },
+        {
+          "Name": "NumberOfNodesUpdate",
+          "CallingType": "string",
+          "Exposure": "public",
+          "_": "u:NumberOfNodes"
+        },
+        {
+          "Name": "getNearestIndexNode",
+          "CallingType": "json",
+          "Exposure": "",
+        }
+      ]
+    },
+    {
+      "Name": "IndexContent",
+      "Description": "Generic indexing functions provision",
+      "NucleusType": "js",
+      "CodeFile": "indexcontent.js",
+      "Entries": [
+        {
+          "Name": "ContentToIndex",
+          "Required": true,
+          "DataFormat": "string",
+          "Sharing": "public",
+          "_": "cru"
```

Comment:
these "_" items should be removed because they are an artifact of the scaffold file, and don't actually belong in the DNA